### ISSUE TITLE
esm_viz: command not found

### ISSUE THREAD

**Issue by chrisdane** [pgierz/esm-viz#15]
After installing, the error
```
$ esm_viz --version
esm_viz: command not found
```
occurs. However, installing probably was successful?:
```
$ pip install --user .
Processing /home/csys/cdanek/esm/esm-viz
  Requirement already satisfied (use --upgrade to upgrade): esm-viz==0.8.0 from file:///home/csys/cdanek/esm/esm-viz in /home/csys/cdanek/.local/lib/python2.7/site-packages
Requirement already satisfied: Click>=6.0 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from esm-viz==0.8.0)
Requirement already satisfied: nbformat in /home/csys/cdanek/.local/lib/python2.7/site-packages (from esm-viz==0.8.0)
Requirement already satisfied: nbmerge in /home/csys/cdanek/.local/lib/python2.7/site-packages (from esm-viz==0.8.0)
Requirement already satisfied: paramiko in /home/csys/cdanek/.local/lib/python2.7/site-packages (from esm-viz==0.8.0)
Requirement already satisfied: python-crontab in /home/csys/cdanek/.local/lib/python2.7/site-packages (from esm-viz==0.8.0)
Requirement already satisfied: pyyaml in /home/csys/cdanek/.local/lib/python2.7/site-packages (from esm-viz==0.8.0)
Requirement already satisfied: traitlets>=4.1 in /usr/local/lib/python2.7/dist-packages (from nbformat->esm-viz==0.8.0)
Requirement already satisfied: ipython-genutils in /usr/local/lib/python2.7/dist-packages (from nbformat->esm-viz==0.8.0)
Requirement already satisfied: jupyter-core in /home/csys/cdanek/.local/lib/python2.7/site-packages (from nbformat->esm-viz==0.8.0)
Requirement already satisfied: jsonschema!=2.5.0,>=2.4 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from nbformat->esm-viz==0.8.0)
Requirement already satisfied: bcrypt>=3.1.3 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from paramiko->esm-viz==0.8.0)
Requirement already satisfied: pynacl>=1.0.1 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from paramiko->esm-viz==0.8.0)
Requirement already satisfied: cryptography>=2.5 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from paramiko->esm-viz==0.8.0)
Requirement already satisfied: python-dateutil in /home/csys/cdanek/.local/lib/python2.7/site-packages (from python-crontab->esm-viz==0.8.0)
Requirement already satisfied: decorator in /home/csys/cdanek/.local/lib/python2.7/site-packages (from traitlets>=4.1->nbformat->esm-viz==0.8.0)
Requirement already satisfied: six in /home/csys/cdanek/.local/lib/python2.7/site-packages (from traitlets>=4.1->nbformat->esm-viz==0.8.0)
Requirement already satisfied: enum34; python_version == "2.7" in /usr/local/lib/python2.7/dist-packages (from traitlets>=4.1->nbformat->esm-viz==0.8.0)
Requirement already satisfied: attrs>=17.4.0 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from jsonschema!=2.5.0,>=2.4->nbformat->esm-viz==0.8.0)
Requirement already satisfied: functools32; python_version < "3" in /usr/local/lib/python2.7/dist-packages (from jsonschema!=2.5.0,>=2.4->nbformat->esm-viz==0.8.0)
Requirement already satisfied: setuptools in /home/csys/cdanek/.local/lib/python2.7/site-packages (from jsonschema!=2.5.0,>=2.4->nbformat->esm-viz==0.8.0)
Requirement already satisfied: pyrsistent>=0.14.0 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from jsonschema!=2.5.0,>=2.4->nbformat->esm-viz==0.8.0)
Requirement already satisfied: cffi>=1.1 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from bcrypt>=3.1.3->paramiko->esm-viz==0.8.0)
Requirement already satisfied: asn1crypto>=0.21.0 in /home/csys/cdanek/.local/lib/python2.7/site-packages (from cryptography>=2.5->paramiko->esm-viz==0.8.0)
Requirement already satisfied: ipaddress; python_version < "3" in /home/csys/cdanek/.local/lib/python2.7/site-packages (from cryptography>=2.5->paramiko->esm-viz==0.8.0)
Requirement already satisfied: pycparser in /home/csys/cdanek/.local/lib/python2.7/site-packages (from cffi>=1.1->bcrypt>=3.1.3->paramiko->esm-viz==0.8.0)
You are using pip version 9.0.1, however version 19.2.3 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
```

**Comment by pgierz** [comment]
So, 

```
esm_viz: command not found
```
that comes from the fact that your shell isn't including `pip`'s basic location for the `--user` option as part of it's `$PATH`

Ideally I'd like to just do `pip install esm_viz` without any of the `git clone` stuff before hand, but I need to understand how Python is actually doing it's packaging first. Time for a technical youtube evening.....

**Comment by chrisdane** [comment]
```
$ ./esm_viz --version
Traceback (most recent call last):
  File "./esm_viz", line 11, in <module>
    load_entry_point('esm-viz==0.8.1', 'console_scripts', 'esm_viz')()
  File "/home/csys/cdanek/.local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/home/csys/cdanek/.local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2852, in load_entry_point
    return ep.load()
  File "/home/csys/cdanek/.local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2443, in load
    return self.resolve()
  File "/home/csys/cdanek/.local/lib/python2.7/site-packages/pkg_resources/__init__.py", line 2449, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/home/csys/cdanek/.local/lib/python2.7/site-packages/esm_viz/cli.py", line 19, in <module>
    from esm_viz.deployment import Simulation_Monitor
ImportError: No module named deployment
```

**Comment by pgierz** [comment]
Closed by 75a4d8d2b9a42a461a21d63ab0f5d7bb17e1588d
